status-im/status-react
# Implement Message Context Drawer

fixes #13527 

### Summary

Implementation of Message Context Drawer

![contextdrawers](https://user-images.githubusercontent.com/18485527/187287536-06ed301d-5bf4-4497-984b-a1815c988122.gif)

### Testing notes

- Some of the cases depend on the implementation of other features (GIF messages, multiple image message, etc.) so those will be implemented separately
- Although there are cases that can be implemented, some of the options for those cases are not implemented yet so I am not showing them, will be implemented separately
- Created issues for all those things that were left out and assigned them a tag: https://github.com/status-im/status-mobile/labels/message-context-drawer

#### Platforms

- Android
- iOS

##### Functional

- 1-1 chats
- public chats
- group chats

### Steps to test

- Open Status
- Log in
- Enter a chat
- Long-press on different types of messages in different contexts to see the Message Context Drawer

status: ready
